public/app/percona/settings/Settings.messages.ts

```diff
@@ -13,6 +13,8 @@ export const Messages = {
     telemetryLabel: 'Telemetry',
     telemetryLink: `${GUI_DOC_URL}#server-admin-gui-telemetry`,
     telemetryTooltip: 'Option to send usage data back to Percona to let us make our product better.',
+    telemetryDisclaimer:
+      'Disabling Telemetry while Integrated Alerting/STT is on will inactivate intelligence (Downloading check and Alert templates).',
```

Comment:
Is "Integrated Alerting" correct here or should it be just "Alerting"?

Reply:
Yep, shouldn't be :D